aspnetcore/test/integration-tests/includes/integration-tests9.md

```diff
@@ -66,8 +66,22 @@ Test classes implement a *class fixture* interface ([`IClassFixture`](https://xu
 The following test class, `BasicTests`, uses the `WebApplicationFactory` to bootstrap the SUT and provide an <xref:System.Net.Http.HttpClient> to a test method, `Get_EndpointsReturnSuccessAndCorrectContentType`. The method verifies the response status code is successful (200-299) and the `Content-Type` header is `text/html; charset=utf-8` for several app pages.
 
 <xref:Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory%601.CreateClient> creates an instance of `HttpClient` that automatically follows redirects and handles cookies.
-  
-[!code-csharp[](~/../AspNetCore.Docs.Samples/test/integration-tests/9.x/IntegrationTestsSample/tests/RazorPagesProject.Tests/IntegrationTests/BasicTests.cs?name=snippet1)]
+
+:::zone pivot="xunit"
+
+:::code language="csharp" source="../snippets/xunit/IntegrationTests/BasicTests.cs?name=snippet1":::
```

Comment:
I think this is easier to read and recommended, I could be wrong though.

```suggestion
:::code language="csharp" source="../snippets/xunit/IntegrationTests/BasicTests.cs" id="snippet1":::
```

Reply:
Agreed